packages/snaps-controllers/src/services/node/NodeProcessExecutionService.test.ts

```diff
@@ -68,8 +69,25 @@ describe('NodeProcessExecutionService', () => {
           params: {},
           id: 1,
         },
-      }),
-    ).rejects.toThrow('foobar');
+      })
+      .catch((error) => error);
+
+    expect(result).toBeInstanceOf(JsonRpcError);
+
+    // @ts-expect-error - This is a `JsonRpcError`.
+    // eslint-disable-next-line jest/prefer-strict-equal
```

Comment:
Why are these lines needed?

Reply:
TypeScript can't infer the type of errors (since anything is technically a valid error). Jest's `.toBeInstanceOf` also doesn't do any type-level assertions. I'm not sure why we need `.toEqual` here, but the tests fail with `.toStrictEqual`.